### DISCUSSION TITLE
BRESSER CH5 Temperature Sensor identified as HIDEKI TS04 Sensor

### DISCUSSION THREAD

**Discussion by sriccardi-invn** [merbanan/rtl_433#2345] [General]
BRESSER CH2 is in the list of Sensor comaptible.
I have a BRESSER CH5, Tmperature and Humidity Sensor.
rtl_433 identified BRESSER CH5 as protocol 42  # HIDEKI TS04 Temperature, Humidity, Wind and Rain Sensor.
Temperature and Humidity valkue are correct:

time      : 2023-01-24 17:23:00
model     : HIDEKI TS04 sensor                     Rolling Code: 0           Channel   : 1             Battery   : OK
Temperature: 15.3 C      Humidity  : 31 %          MIC       : CRC

**Comment by zuckschwerdt** [comment]
This will not happen. "Bresser 3CH", "Bresser 5in1", "Hideki-TS04" are *very* different protocols.

Maybe someone in your neighbourhood has a Hideki-TS04?
Maybe you don't have a "Bresser 3CH" or "Bresser 5in1"?
(or whatever you mean with "BRESSER CH2" / "BRESSER CH5"?)

**Comment by sriccardi-invn** [comment]
Sure that this is the Bresser, same temperature and Humidity.
More than this when the red led blick on the device, I receive the frame just 1 sec after.
BRESSER Chip is Thermo-Hydro Sensor 5CH Art N° 7009993
![Uploading WIN_20230124_20_31_33_Pro.jpg…]()

**Reply by zuckschwerdt** [reply]
OK, thanks. So the "Bresser 5CH" (7009993) is a rebrand of the Hideki-TS04 :) We'll add a note.
Bresser make lots of rebrands.
The "Bresser 3CH" and e.g. "Bresser 5in1" are very different. The "CH" or "5" does not mean anything here.

**Reply by sriccardi-invn** [reply]
I confirmed that in this device you can select 5 Chanels with a switch.
